packages/flex-dev-utils/src/validators.ts

```diff
@@ -0,0 +1,91 @@
+// tslint:disable-next-line
```

Comment:
what's wrong with the next line?

Reply:
It's long; linter complains about the length

Reply:
wat the next line is not long enough for linter to complain

Reply:
Sorry ya wrong place - fixiing it now